platform/src/apis/Platform.Api.Establishment/Features/Trusts/Functions/GetTrustFunction.cs

```diff
@@ -1,36 +1,37 @@
 using System.Net;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Azure.Functions.Worker.Http;
 using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+using Platform.Api.Establishment.Features.Trusts.Handlers;
 using Platform.Api.Establishment.Features.Trusts.Models;
-using Platform.Api.Establishment.Features.Trusts.Services;
 using Platform.Functions;
 using Platform.Functions.Extensions;
 using Platform.Functions.OpenApi;
 
-namespace Platform.Api.Establishment.Features.Trusts;
+namespace Platform.Api.Establishment.Features.Trusts.Functions;
 
-public class GetTrustFunction(ITrustsService service)
+public class GetTrustFunction(IVersionedHandlerDispatcher<IGetTrustHandler> dispatcher)
 {
     [Function(nameof(GetTrustFunction))]
     [OpenApiSecurityHeader]
     [OpenApiOperation(nameof(GetTrustFunction), Constants.Features.Trusts)]
     [OpenApiParameter("identifier", Type = typeof(string), Required = true)]
     [OpenApiResponseWithBody(HttpStatusCode.OK, ContentType.ApplicationJson, typeof(Trust))]
```

Comment:
It would be handy to also include the optional header in the affected OpenApi specs, e.g.:

```csharp
[OpenApiParameter(Platform.Functions.Constants.ApiVersion, Type = typeof(string), Required = false, In = ParameterLocation.Header)]
```